include/util/coordinate.hpp

```diff
@@ -40,7 +40,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 namespace osrm
 {
 
-constexpr const double COORDINATE_PRECISION = 1e6;
+constexpr const double COORDINATE_PRECISION = 1e7;
```

Comment:
I think this could break existing user-code since our public interface include these private headers (which is unfortunate on its own):

https://github.com/Project-OSRM/osrm-backend/blob/1ae69fe40d68a15ca978f0a4535d75523b292467/include/osrm/coordinate.hpp#L31